proto/internal/temporal/server/api/persistence/v1/executions.proto

```diff
@@ -223,6 +223,14 @@ message TimerTaskInfo {
     int64 task_id = 10;
     google.protobuf.Timestamp visibility_time = 11 [(gogoproto.stdtime) = true];
     bytes branch_token = 12;
+    message DeleteHistoryEventTaskDetails {
```

Comment:
Could it just be simple as bool already_archived = 13 ?

Reply:
It definitely could be. My idea here was that this is only specific to `DeleteHistoryEventTask`, so that makes it more obvious what its purpose is. For example, if a future dev is working on user timer tasks, they might interpret this flag to mean the data hasn't been archived, but it actually means nothing in that context. However, I can change this to an inline field if you'd like.

Reply:
Will remove and do inline

Reply:
Now inline